vnext/JSI/Shared/ChakraRuntime.h

```diff
@@ -248,15 +239,35 @@ class ChakraRuntime : public facebook::jsi::Runtime {
   ChakraObjectRef ToChakraObjectRef(const facebook::jsi::Value &value);
   std::vector<ChakraObjectRef> ToChakraObjectRefs(const facebook::jsi::Value *value, size_t count);
 
-  // Host function and host object helpers
+  // Convenience functions for property access.
+  ChakraObjectRef GetProperty(const ChakraObjectRef &obj, const ChakraObjectRef &id);
+  inline ChakraObjectRef GetProperty(const ChakraObjectRef &obj, const char *const name) {
+    return GetProperty(obj, GetChakraObjectRef(createPropNameIDFromAscii(name, strlen(name))));
```

Comment:
>createPropNameIDFromAscii [](start = 47, length = 25)

(nit) similarly, use "id" or "name" consistently.

Reply:
Same as previous comment.

---
In reply to: [341262364](https://github.com/microsoft/react-native-windows/pull/3568#discussion_r341262364) [](ancestors = 341262364)